### PR TITLE
Support stl without assimp

### PR DESCRIPTION
**Pull request by taiki-e**
~~Based on #81~~

### CODE REVIEW

File: src/utils.rs [hunk 1]
```diff
@@ -71,27 +74,83 @@ mod wasm {
...
+            match &self.data {
+                Data::None => None,
+                Data::Bytes(s) => str::from_utf8(s).ok(),
+                Data::Base64(..) => unreachable!(),
+            }
+        }
+    }
+
+    #[derive(Serialize, Deserialize)]
+    pub(crate) enum Data {
```
Comment (OTL) [inline src/utils.rs:111]: Data is not a good name generally.
Could you add some information to this, like `MeshData` if this were only for meshes? (If you agree with this)

File: src/utils.rs [hunk 2]
```diff
@@ -127,13 +186,22 @@ mod wasm {
                             .unwrap()
                             .to_string()
                     };
-                let (kind, data) = if input_file.ends_with(".obj") {
+
+                let kind = if input_file.ends_with(".obj") {
+                    MeshKind::Obj
+                } else if input_file.ends_with(".stl") || input_file.ends_with(".STL") {
```
Comment (OTL) [inline src/utils.rs:192]: Same as above. .OBJ is also possible.

Comment (taiki-e) [reply src/utils.rs]: added support of "OBJ"

File: src/mesh.rs [hunk 3]
```diff
@@ -99,26 +102,38 @@ pub fn load_mesh(
...
+                mtl_path.display()
+            );
+            let mut base = group.add_obj(filename, mtl_path, scale);
+            if let Some(color) = *opt_color {
+                base.set_color(color[0], color[1], color[2]);
+            }
+            Ok(base)
         }
-        return Ok(base);
+        Some("stl") | Some("STL") => {
```
[the remark this answers is not in the document]
Comment (taiki-e) [reply src/mesh.rs:123]: added support of "OBJ"

File: src/utils.rs (hunk 1 above)
Comment (taiki-e) [reply src/utils.rs]: renamed to MeshData

**Review by OTL — APPROVED** [review]

**Pull request merged**